speech/google/cloud/speech_v1/proto/cloud_speech.proto

```diff
@@ -752,4 +744,12 @@ message WordInfo {
 
   // The word corresponding to this set of information.
   string word = 3;
+
+  // A distinct integer value is assigned for every speaker within
+  // the audio. This field specifies which one of those speakers was detected to
+  // have spoken this word. Value ranges from '1' to diarization_speaker_count.
+  // speaker_tag is set if enable_speaker_diarization = 'true' and only in the
+  // top alternative.
+  int32 speaker_tag = 5
+    [(google.api.field_behavior) = OUTPUT_ONLY];
 }
```

Comment:
This is potentially a breaking change, although it looks obviously correct (the unchanged docstring implies that the `speaker_tag` should always have been associated with a word).

The field was [added to `SpeakerDiarizationConfig`](https://github.com/googleapis/googleapis/commit/62167121209eabec747206f32174b71f6654db27#diff-26922cad25fec0fba17dae424d1c8c8b) on 2019-09-24, which is later than any release of `google-cloud-speech`, so I think we can let it slide.

Reply:
This may be reverted tomorrow given discussions in the Yoshi chatroom. Will be keeping an eye on it.